sdk/core/azure-core/CMakeSettings.json

```diff
@@ -0,0 +1,44 @@
+﻿{
+  "configurations": [
```

Comment:
But we have the same file checked in for the root directory. If we keeping that, let's have this one too. Because it is convenient to load just a single directory, build, and verify that it builds in isolation, which is important for vcpkg.
Not the end of the world, this file can be deleted and recreated, but I think then the same file in the root directory needs to go as well. But I think having this file is rather a convenience. Let me know, I have no strong opinion on this.